### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Eccb7**
In the mobile version, I've implemented the following features:

-When the user clicks (or taps) the hamburger button, the mobile menu appears.
-When the user clicks (or taps) the close (X) button, the mobile menu disappears.
-When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
-When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by koachcode — COMMENTED** [review]
Hi @Eccb7 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that requires pair programming, however i can only see one person committing to the project instead of more than one.  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -51,6 +51,60 @@ a {
...
+    width: 100%;
+    height: 100vh;
+    background-color: #6070ff;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(8px);
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    z-index: 999;
+  }
```
Comment (AmaduKamara) [inline style.css]: - [ ] The mobile menu is expected to be fixed and not scrolled. Currently, when I scroll on mobile view, the menu scrolls along. It's expected to stay fixed and only closed when an item in the list or the close(X) button is clicked. Could you work on making the mobile menu fixed and not scrollable? 👍🏽
- [ ] Also, please align the mobile menu navigation list items to the left just as it is in the original design. 👍🏽


**Your Design**

Screenshot 2023-05-17 at 8 17 40 AM

**The Original Design**

Screenshot 2023-05-17 at 8 30 04 AM

**Note: The list items are aligned to the left and not center**

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @Eccb7,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Good commit messages 👍🏽

## Required Changes ♻️
_Check the comments under the review._

✅ Kindly work on making the mobile menu fixed and not scrollable. It closes or is not visible only when the X close icon is clicked or when an item is clicked but must not be scrollable. 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by wayungi — APPROVED** [review]
Hi @Eccb7 , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ]  Fixed requested changes ✔️ 
- [ ] All links on mobile menu are working ✔️ 
- [ ] Hamburger is not visible on desktop ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**